packages/good-design/src/core/layout/BalanceGD.tsx

```diff
@@ -2,18 +2,25 @@ import React, { FC, memo } from "react";
 import { Text, View } from "native-base";
 import { useG$Balance } from "@gooddollar/web3sdk-v2";
 import { Fraction } from "@uniswap/sdk-core";
-import { CurrencyValue } from "@usedapp/core";
+import { CurrencyValue, QueryParams } from "@usedapp/core";
 
 interface BalanceGDProps {
   gdPrice?: Fraction;
+  refresh?: QueryParams["refresh"];
 }
 
+const BalanceGD: FC<BalanceGDProps> = ({ gdPrice, refresh = "never" }) => {
```

Comment:
let's add **optional** `requiredChainId?: number` here too

```suggestion
const BalanceGD: FC<BalanceGDProps> = ({ gdPrice, requiredChainId, refresh = "never" }) => {
```